examples/gpt/schedule.py

```diff
@@ -5,8 +5,7 @@
 
 import torch
 import torch.nn as nn
-import torch.distributed as dist
-from torch.distributed.distributed_c10d import _get_global_rank
+import torch.distributed.distributed_c10d as dist
```

Comment:
Is this the same as `torch.distributed`?
Also it is recommended to use `from torch import xxx as dist` instead.

Reply:
changed the import style